src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1687,6 +1742,7 @@ components:
             - none
             - read_only
             - read_write
+            - clearing_admin
             - admin
         rootFolderId:
```

Comment:
Can you please add `default_group_fk` from `users` table as well to the model, preferable called as `defaultGroup`?

It denotes the default group selected by the user.

Doing so, it will be available from `GET /api/v1/users[/self|{id}]` endpoint as well and while `PUT /api/v1/users/{id}`, it can be read to update the default group id for the user.

Reply:
You mean, adding a parameter to the `User` model which will be called `defaultGroup` so as to allow the admin to edit the default group of the user? right?

Reply:
I have done the change and updated the PR. Please have a look.